cpp/proto/serializer_base.h

```diff
@@ -0,0 +1,94 @@
+#ifndef CERT_TRANS_PROTO_SERIALIZER_BASE_H_
+#define CERT_TRANS_PROTO_SERIALIZER_BASE_H_
+
+#include <glog/logging.h>
+#include <string>
+
+#include "proto/ct.pb.h"
+
+namespace serialization {
```

Comment:
I think you should have this inside the `cert_trans` namespace as well (making it overall `cert_trans::serialization`).